db/migrate/20190530033104_add_username_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddUsernameToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :username, :string, limit: 30
```

Comment:
You might want to also enforce this on the model level with something like `validates :username, length: { maximum: 30 }`

Reply:
Added the suggested validation within `user.rb`. Thank you :smile: